### DISCUSSION TITLE
Batterie "blockieren" für E-Auto Laden aus dem Netz

### DISCUSSION THREAD

**Discussion by Mondushiva** [weltenwort/home-assistant-rct-power-integration#393] [Ideas]
Hallo,

erstmal vielen Dank für diesen HACS. Ich konnte bereits ein PV-Überschussladen für meine Wallbox damit realisieren und das funktioniert ausgezeichnet. Eine Idee die mich umtreibt: Ist es möglich die Batterie zu "blockieren" um mein E-Auto gezielt aus dem Netz zu laden (z.B. im Winter oder bei schlechtem Wetter) ohne dabei zuerst die (Haus-)Batterie entleeren zu müssen? Vielleicht ist es möglich den SOC zu fixieren? 

Ich lese das es an Dokumentation seitens RCT fehlt. Auf die Frage nach der Sunspec-API habe ich recht umfangreiche PDFs von RCT zugesandt bekommen und ich frage mich ob diese hier vorliegen? Ich verstehe von deren Inhalt leider nahezu Garnichts aber würde diese gerne zur Verfügung stellen in der Hoffnung, dass der Arbeit am HACS nutzt.
[RCT Power Serial Communication Protocol v1.14 (2).pdf](https://github.com/user-attachments/files/15826924/RCT.Power.Serial.Communication.Protocol.v1.14.2.pdf)
[RCT_Power_RS485_Communication_20200429_v.1.2 (6).pdf](https://github.com/user-attachments/files/15826925/RCT_Power_RS485_Communication_20200429_v.1.2.6.pdf)

**Comment by do-gooder** [comment]
Ich arbeite da gerade an etwas:
https://github.com/do-gooder/rctpower_writesupport
